bin/update-cesm-machines

```diff
@@ -196,12 +196,13 @@ def main():
         'external_path', default=None, help="Path to external directory with extra XML files",
     )
     cli.add_argument(
-        'machine_files',
+        '-m', '--machine-files',
         help=(
             "CESM machine files to process. Known machine files are batch, compilers and machines. "
             "Option 'other' allows to update any arbitrary machine file."
         ),
         choices=['batch', 'compilers', 'machines', 'other'],
+        default=['batch', 'compilers', 'machines'],
```

Comment:
users can only specify one machine file, but the default is 3 machine files.
is this intended? or should users be able to specify multiple machine files?